lms/static/js/learner_dashboard/models/course_enroll_model.js

```diff
@@ -0,0 +1,97 @@
+/**
+ *  Track optIn status and submit form
+ */
+;(function (define) {
+    'use strict';
+
+    define([
+            'backbone',
+            'jquery',
+            'underscore'
+        ],
+        function( Backbone, $, _ ) {
+            return Backbone.Model.extend({
+                defaults: {
+                    course_id: '',
+                    optIn: false,
+                    enrolled: false,
+                },
+
+                initialize: function( attributes, options ) {
+                    var api = '/api/commerce/v0/baskets/';
+
+                    this.set({
+                        course_id: options.courseId,
+                    });
+
+                    this.url = api;
+                    this.trackSelectionUrl = '/course_modes/choose/';
+                    this.dashboardUrl = '/dashboard';
+                    this.verificationUrl = '/verify_student/start-flow/';
```

Comment:
Are you saying, I should be passing these url in from the lms backend instead of hard coding them in javascript?